guides/v2.2/cloud/project/project-conf-files_services-mysql.md

```diff
@@ -6,40 +6,40 @@ functional_areas:
   - Setup
 ---
 
-The `mysql` service provides data storage. It is based on [MariaDB](https://mariadb.com/), supporting the [XtraDB](https://www.percona.com/software/mysql-database/percona-server/xtradb) storage engine (equivalent to MySQL with InnoDB).
+The `mysql` service provides persistent data storage. It is based on [MariaDB](https://mariadb.com/), supporting the [XtraDB](https://www.percona.com/software/mysql-database/percona-server/xtradb) storage engine.
 
-We support MariaDB version 10.0, which includes reimplemented features from MySQL 5.6 and 5.7.
+We support MariaDB version 10.2, which includes reimplemented features from MySQL 5.6 and 5.7.
 
-To access the MariaDB database directly, [open an SSH tunnel]({{ page.baseurl }}/cloud/env/environments-start.html#env-start-tunn) and use the following command:
+Using SSH, log in to the remote server and access the MariaDB database directly:
 
 ```bash
 mysql -h database.internal -u user
```

Comment:
It will be work only for starter. For PRO customer should take login, password and DB name from RELATIONSHIP